README.md

```diff
@@ -1,2 +1,73 @@
-# koa-npm-resolution
-Koa middleware that transforms NPM package specifiers to relative paths
+# koa-node-resolve
+
+Middleware for Koa servers that resolves Node package specifiers in standard JS modules to relative paths for use on the web.
+
+The following import uses a _bare module specifier_, which won't currently load natively in browsers (until [import maps](https://www.chromestatus.com/feature/5315286962012160) are available):
+
+```js
+import { foo } from "stuff";
+```
+
+`koa-node-resolve` solves this problem by resolving `stuff` using the same rules as [Node `require()`](https://nodejs.org/api/modules.html#modules_all_together), and transforming the import specifier to a path that can be loaded natively by any browser that [supports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Browser_compatibility) standard JS modules:
+
+```js
+import { foo } from "./node_modules/stuff/index.js";
+```
+
+Because this is middleware, you can use it in a simple static file server as well as a proxy server sitting in front of a test server such as the one `karma` starts up. (See [karma testing setup](#karma-testing-setup) below.)
+
+Note: HTML and JavaScript are parsed on every request for those content-types, it is intended for use in development context to facilitate build-free testing/iteration as opposed to in a high volume production web server.
+
+## Installation
+
+```sh
+$ npm install --save koa-node-resolve
+```
+
+## Usage
+
+Create your own mini-development server in file `./dev-server.js`. This one depends on `koa` and `koa-static`, so you'll need to `npm install --save-dev koa koa-static` for your project to use it.
+
+```js
+const Koa = require("koa");
+const server = new Koa()
+  .use(require("koa-node-resolve").middleware())
```

Comment:
update this

Reply:
Updated.  Made these named variables before they're used.